.kokoro/stage.sh

```diff
@@ -49,13 +43,13 @@ create_settings_xml_file $MAVEN_SETTINGS_FILE
   --activate-profiles skip-unreleased-modules
 
 # promote release
-if [[ -n "${AUTORELEASE_PR}" ]]
-then
-  ./mvnw nexus-staging:release \
-    --batch-mode \
-    --settings ${MAVEN_SETTINGS_FILE} \
-    -Drelease=true \
-    --activate-profiles skip-unreleased-modules
-fi
-
-popd
+#if [[ -n "${AUTORELEASE_PR}" ]]
```

Comment:
Thanks @ddixit14! Probably on your radar already, but dropping a note to uncomment this back when merging 😄